content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,101 @@
+
+### Upgrading from older Kubernetes versions
+
+If you are upgrading from a Kubernetes release older than 1.26, and you are on AWS, Azure, GCE, OpenStack, or vSphere then you will need to enable the `--cloud-provider=external` flag, and follow the advice for installing and running a cloud controller manager for your provider.
+
+Please read the documentation for [Cloud Controller Manager Administration][ccmadmin] and [Migrate Replicated Control Plane To Use Cloud Controller Manager][ccmha]. See below for links to specific cloud provider implementations.
+
+## Where to find a cloud controller manager?
+
+At its core, this post is about the cloud controllers that were previously included in Kubernetes. As these components move out of the core Kubernetes code and into their own repositories it is important to note a few things.
+
+First, SIG Cloud Provider maintains a reference framework for developers who wish to create cloud controller managers for any provider. See the [cloud-provider repository][cloud-provider] for more information about how these controllers work and how to get started creating your own.
+
+Second, there are many cloud controller managers available for Kubernetes. This post is addressing the providers that have been historically included with Kubernetes but are now in the process of being removed. If you need a cloud controller manager for your provider and do not see it listed here, please reach out to the SIG Cloud Provider community (links below) for help and advice.
+
+### Cloud Controller Managers
+
+* AWS - https://github.com/kubernetes/cloud-provider-aws
+* Azure - https://github.com/kubernetes-sigs/cloud-provider-azure
+* GCE - https://github.com/kubernetes/cloud-provider-gcp
+* OpenStack - https://github.com/kubernetes/cloud-provider-openstack
+* vSphere - https://github.com/kubernetes/cloud-provider-vsphere
+
+## Want to learn more?
+
+Cloud providers and cloud controller managers serve a core function in Kubernetes. Cloud providers are often the substrate upon which Kubernetes is operated, and the cloud controller managers supply the essential lifeline between Kubernetes clusters and their physical infrastructure.
+
+This post exposes one aspect of how the Kubernetes community interacts with the world of cloud infrastructure providers. If you are curious about this topic and want to learn more the Cloud Provider Special Interest Group (SIG) is the place to go. SIG Cloud Provider hosts bi-weekly meetings to discuss all manner of topics related to cloud providers and cloud controller managers in Kubernetes.
+
+### SIG Cloud Provider
+
+* Regular SIG Meeting: [Wednesdays at 9:00 PT (Pacific Time)](https://zoom.us/j/508079177?pwd=ZmEvMksxdTFTc0N1eXFLRm91QUlyUT09) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+* [Kubernetes slack][kslack] channel `#sig-cloud-provider`
+* [SIG Community page][sig]
```

Comment:
Although this might feel like a feature, readers will experience this more as a loss: they didn't have to do a thing, and now they do. I would omit these details given we're basically imposing work on end users.

Reply:
just to be clear, remove the SIG details or the entire section on "Want to learn more" ?